packages/flutter/test/widgets/scroll_view_test.dart

```diff
@@ -128,6 +128,41 @@ void main() {
     expect(textField.focusNode.hasFocus, isFalse);
   });
 
+  testWidgets('GridView dismiss keyboard onDrag test', (WidgetTester tester) async {
```

Comment:
We need test for each constructor as well

Reply:
As I see,  arguments are refer to BoxScrollView,  is it a duplicate test code for that?